test/e2e/carousel.test.js

```diff
@@ -75,6 +44,63 @@ describe('Nuka Carousel', () => {
 
       await expect(hiddenStyles.visibility).toMatch('hidden');
     });
+
+    describe('when in wrapAround mode', () => {
```

Comment:
I moved these existing tests into a separate describe so each `it` statement didn't have to say "when wrapAround is true"